go/kbfs/libpages/cert_store_backed_by_kvstore.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2020 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package libpages
+
+import (
+	"context"
+	"encoding/base64"
+
+	"github.com/keybase/client/go/kbfs/libkbfs"
+	"github.com/keybase/client/go/protocol/keybase1"
+	"github.com/pkg/errors"
+	"golang.org/x/crypto/acme/autocert"
+)
+
+// certStoreBackedByKVStore is a wrapper around the KVStore in Keybase. No
+// caching is done here since acme/autocert has a in-memory cache already.
+type certStoreBackedByKVStore struct {
+	kbfsConfig libkbfs.Config
```

Comment:
Maybe just pass in and store the kvstoreclient here?  Would be better for testing, if we ever add tests for it.

Reply:
I was a bit worried that something along `s.kbfsConfig.KeybaseService().GetKVStoreClient()` might be switched out when the process is running, but after a few greps it looks like they never change once init'ed, so it's probably OK I guess. Could you confirm?

Reply:
Looking at this more, it seems `kbpki_client.go` always calls `KeybaseService()`: https://github.com/keybase/client/blob/ea7eea5070a2dd74a75694352c2f385bfa0945ba/go/kbfs/libkbfs/kbpki_client.go#L59 so perhaps this should too? I guess we could define a service owner here too like in kbpki.